distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -1255,7 +1255,8 @@ This plugin can be installed and uninstalled using the normal processes describe
 No configuration is necessary for this plugin. 
 By default, it will mark metacards with validation errors and warnings as they are reported by each metacard validator and then allow the ingest.
 To prevent the ingest of certain invalid metacards, the `Metacard Validity Marker` plugin can be configured to "enforce" one or more validators.
-Metacards that are invalid according to an "enforced" validator will not be ingested.
+The `Metacard Validity Marker` plugin can also be configured to enforce validator errors, warnings, or both.
+Metacards that are have enforced errors or enforced warnings on enforced validators will not be ingested.
```

Comment:
It's still not quite clear to me (but that could just be me.) I'm having trouble understanding the matrix of 
`enforced validator` & `unenforced validator` X `enforced error` & `unenforced error`

I don't know what an `unenforced error` + `enforced validator` means. What makes that error unenforced in that case?

Not knowing how this is all supposed to fit, this is what I would assume:
1. Errors/warnings can be reported from any validator
2. If errors/warnings are reported from a validator that is not marked as enforced, they will only be reported.
3. If errors/warnings are reported from a validator marked as enforced, but the errors/warnings are themselves not enforced, they will only be reported.
4. If errors/warnings are reported from a validator that is marked as enforced, _and_ they are themselves marked as enforced, they will prevent the ingest.

Is that a correct interpretation of the rules here?